compiler/codegen/codegen.cpp

```diff
@@ -2672,7 +2672,7 @@ GenInfo::GenInfo()
              functionCNameAstrToSymbol()
 #ifdef HAVE_LLVM
              ,
-             lvt(NULL), module(NULL), irBuilder(NULL), mdBuilder(NULL),
+             lvt(nullptr), module(NULL), irBuilder(NULL), mdBuilder(NULL),
```

Comment:
I'm kindof baffled why this PR changes just one of these on the line

Reply:
otherwise the constructor for `unique_ptr` is ambiguous

Reply:
Ah, I was imagining you were updating it out of a general practice of using `nullptr` instead of `NULL`. Anyway it seems to me that you might as well change the other NULLs in this initialization list to nullptr.

Reply:
Okay I went ahead and made that change. I tend to err on the side of fewer incidental change